process/OpenHW-Specification-Process.md

```diff
@@ -0,0 +1,177 @@
+| :---		| :------ 									| :-------------------------------------------------------------------------------- |
+| 1 		| Specification published by				| OpenHW Group on Github and/or ReadtheDocs and/or website								|
+| 2 		| Copyright holder on specification			| OpenHW Group on behalf of contributors (member), who hold the copyright jointly. 		|
+| 3			| License used for publication 				| Same license as used for contributions 											|
+| 4			| Copyright Grant included in specification publication 	 | As per publication license:										|	
+| 5.1		| Apache 2.0								| Yes, copyright grant included with OpenHW Group publication 							|
+| 5.2		| Solderpad 2.1								| Yes, copyright grant included with OpenHW Group publication 							|
+| 5.3		| Creative Commons Attribution Share Alike 4.0 International license | Yes, copyright grant included with OpenHW Group publication 			|								
+| 6			| Patent Grant included in specification publication 	|	As per publication license											|
+| 6.1		| Apache 2.0								| No patent grant provided by OpenHW Group but patents rights granted under contribution	|
+| 6.2		| Solderpad 2.1								| No patent grant provided by OpenHW Group but patents rights granted under contribution	|
+| 6.3		| Creative Commons Attribution Share Alike 4.0 International license | No patent grant included with OpenHW Group publication					|								
+| 7			| Essential patents disclosed by OpenHW Group		| No																				|
+| 8 		| License text included within the specification text |  No - referall only 													|
+| 0			| "No-warranty" or disclaimer				| Required																|
+
+
+### Specification Version numbers
+
+OpenHW Group Specifications shall use semantic versioning https://semver.org/ with the version in the form X.Y.Z.
+The version number is combined with the Specification State (below), for example "1.0.0 - Released" to indicate both the version number and state.
+
+
+
+### Specification States
+
+The ratification or completion state is meant to be written in each OpenHW Group specification so that a reader will be aware of the state of completion of the Specification
+These are the OpenHW Group Specification States:
+
+- “OpenHW Group Specification - In Development"
+- "OpenHW Group Specification - In Review"
+- "OpenHW Group Specification - Release Candidate"
+- "OpenHW Group Specification - Released"
+
+#### "OpenHW Group Specification - In Development"
+
+When the specification is initiated, all drafts will be labelled as “OpenHW Group Specification - In Development"
+(During initial development of the spec before first release, the version number is of the form X.Y.Z with X=0.) 
+
+This state is also used when a released spec is subsequently revised. 
+
+During revision of a released specifiation, the version number increments as appropriate depending on whether the revision is a major or minor update. 
+
+#### "OpenHW Group Specification - In Review"
```

Comment:
I am not sure that this state is practical. While the work is under development, it is practically always under review by the task group. When the TG is done, it goes to review. Mandating a TG internal review is overkill.